scale/profiler/launch-profiler.sh

```diff
@@ -1,4 +1,4 @@
-#! /bin/sh
```

Comment:
Are we sure bash is available in the environment where the profiled is being launched?

Reply:
We've got `${BASH_SOURCE[0]}` in line 18. That made me think that bash was available.